### ISSUE TITLE
WCAG-pagina's: 2.4.1 Blokken omzeilen tekst aanpassen, qua reperterende blokken

### ISSUE THREAD

**Issue by rianrietveld** [nl-design-system/documentatie#1064]
Aanpassing  https://nldesignsystem.nl/wcag/2.4.1/

Er moet nog expliciet bij, dat de skiplink in dit SC bedoeld is voor repertende blokken in pagina's op dezelfde website.
Dus als het een one-pager is, hoeft er geen skip link in.
Maar als er op de website op elke pagina bijvorbeeld een menu is, dan wel.

De rest is best practice, maar geen eis.

Naar aanleiding van de discussie in Slack met Joke op https://codefornl.slack.com/archives/C07059B0VK7/p1719215871565869
En met Brian op de NLDS-borrel van 26 juni
